amethyst_renderer/src/resources.rs

```diff
@@ -68,11 +68,17 @@ impl ScreenDimensions {
     }
 
     /// Returns the current width of the window.
+    ///
+    /// This is returned as a float for user convenience, as this is typically used with other
+    /// float values.  This will only ever be a positive integer though.
```

Comment:
That ain't true, you're setting it to 0,0 at the beginning.

Reply:
Ah, good catch, thanks.